openedx/core/djangoapps/theming/helpers.py

```diff
@@ -17,3 +18,35 @@ def get_value(val_name, default=None, **kwargs):
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
     return microsite.get_value(val_name, default=default, **kwargs)
+
+
+def get_template_path(relative_path, **kwargs):
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    """
+    return microsite.get_template_path(relative_path, **kwargs)
+
+
+def is_request_in_themed_site():
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+    """
+    return microsite.is_request_in_microsite()
+
+
+def get_themed_template_path(relative_path, default_path, **kwargs):
+    """
+    This is a proxy function to hide microsite_configuration behind comprehensive theming.
+
+    It returns path of the themed template (i.e. relative_path) if theming is enabled AND microsite is disabled
```

Comment:
We should be more clear here about the fact that the theming functionality we're checking for in this function is Stanford theming.  This is what the "USE_CUSTOM_THEME" flag enables.